wolfcrypt/src/curve25519.c

```diff
@@ -51,12 +51,43 @@ const curve25519_set_type curve25519_sets[] = {
     }
 };
 
+static const unsigned char kCurve25519BasePoint[CURVE25519_KEYSIZE] = {9};
+
+/* compute the public key from an existing private key, using bare vectors. */
```

Comment:
Make mention of the endianess of the returned public key buffer and also what int value is returned on success here. The only reason of mentioning endianess for this particular function is because some of the curve25519 functions have an optional big endian or little endian flag.

Reply:
Is that re: `wc_curve25519_make_pub()`, and by "make mention", do you mean in the introductory comment?

Reply:
Yes, just a note in the comment, here at the top of the function implementation. Saying that the public key created and set in the buffer is <type> endineaness.